Makefile

```diff
@@ -323,7 +322,9 @@ test-kubectl-workload: ## Run kubectl queries to get all capx workload related o
 .PHONY: test-e2e
 test-e2e: docker-build-e2e $(GINKGO_BIN) cluster-e2e-templates cluster-templates ## Run the end-to-end tests
 	mkdir -p $(ARTIFACTS)
-	$(GINKGO) -v -trace -tags=e2e -focus="$(GINKGO_FOCUS)" $(_SKIP_ARGS) -nodes=$(GINKGO_NODES) --noColor=$(GINKGO_NOCOLOR) $(GINKGO_ARGS) ./test/e2e -- \
+	$(GINKGO) -v --trace --tags=e2e --label-filter="$(LABEL_FILTERS)" --fail-fast $(_SKIP_ARGS) --nodes=$(GINKGO_NODES) \
+	    --no-color=$(GINKGO_NOCOLOR) --output-dir="$(ARTIFACTS)" --junit-report="junit.e2e_suite.1.xml" \
```

Comment:
consider setting the junit-report via a variable

Reply:
@deepakm-ntnx Please don't resolve without addressing the issue :)

Reply:
@thunderboltsid I dont think this needs to be changed at this time to avoid unnecessary variables

Reply:
idk; customizable report name usually helps keeping the CI-code cleaner as you don't have to make assumptions around magic strings present in makefiles.

Reply:
My main point there is that magic strings tend to replicate across the codebase :slightly_smiling_face: 
sometimes, across codebases. Which is why these things are usually parameterized to give the calling entity control while having a sane default. With that said, we can do that in a separate PR.